packages/components/src/buttons/src/Button.module.css

```diff
@@ -14,10 +18,11 @@
     --hop-Button-primary-background-color-hover: var(--hop-primary-surface-hover);
     --hop-Button-primary-background-color-pressed: var(--hop-primary-surface-active);
     --hop-Button-primary-background-color-disabled: var(--hop-neutral-surface-disabled);
-    --hop-Button-primary-border: 0;
-    --hop-Button-primary-border-hover: 0;
-    --hop-Button-primary-border-pressed: 0;
-    --hop-Button-primary-border-disabled: 0;
+    --hop-Button-primary-border: 0.0625rem solid var(--hop-Button-primary-background-color); /* same color as the background */
```

Comment:
0.0625rem this is used a lot, this could be a variable

Reply:
we wouldn't create a variable for "1px", don't see a reason for 0.0625.